backend/apis/docket-pdf-test-files/get-text.js

```diff
@@ -0,0 +1,26 @@
+const fs = require("fs");
```

Comment:
Now that we have the UI that veronica created, is this still needed? If it is still useful to you or others, then let's keep it. But if Veronica's UI is what is being used to do this, then let's delete this file.

Reply:
I'll delete it.